test/test_integration_management.cxx

```diff
@@ -2866,7 +2867,10 @@ TEST_CASE("integration: analytics index management", "[integration]")
             couchbase::core::operations::management::analytics_get_pending_mutations_request req{};
             auto resp = test::utils::execute(integration.cluster, req);
             REQUIRE_SUCCESS(resp.ctx.ec);
-            REQUIRE(resp.stats[index_name] == 0);
+            // In the Core API the key has the `dataverse.dataset` format
+            auto key = fmt::format("{}.{}", dataverse_name, dataset_name);
+            REQUIRE(resp.stats.count(key) == 1);
```

Comment:
This assertion is failing against 6.6 because the server returns an empty response back for some reason. It seems that a similar issue with server versions < 7.0 was observed in the JVM SDKs as well: [JVMCBC-1075](https://issues.couchbase.com/browse/JVMCBC-1075). The same failure happens with the equivalent Public API test.

(Previously this test was passing because the incorrect key was used so `resp.stats[index_name]` was returning the default value which is 0.)

Reply:
@DemetrisChr lets mask this test and skip it for <7.0